src/panic.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 Joe Wilm, The Alacritty Project Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//! Provide special panic-handling behaviors, often platform-specific.
+
+// Use the default behavior of the other platforms.
+#[cfg(not(windows))]
+pub fn attach_handler() {}
+
+// Install a panic handler that renders the panic in a classical Windows error
+// dialog box as well as writes the panic to STDERR.
+#[cfg(windows)]
+pub fn attach_handler() {
+    extern crate winapi;

```

Comment:
All `extern crate` definitions should be either in `src/main.rs` or `src/lib.rs` since this is called from `src/main.rs` it should be there. This is simply for discoverability and for making sure it's not duplicated. It will require an annotation, but that should be fine.

Reply:
That is already defined in `src/main.rs`, so I have dropped mine.